src/labels_by_user_input.py

```diff
@@ -35,3 +45,26 @@ def labels_by_user_input(event_data, pull):
         if f"/un{LABEL_APPROVE}".lower() in body:
             remove_label(pull=pull, label=LABEL_APPROVE)
             set_commit_status_pending_no_approve(commit=last_commit)
+
+    semver_key = get_semver_user_input(body)
+    if semver_key or semver_label_in_pull_labels(pull):
```

Comment:
can you please do:
```python
if semver_key:
    if not semver_label_in_pull_labels(pull):
        add_remove_labels()
    last_commit.create_status()
```
To avoid call `if semver_key` twice.

Reply:
We cannot do this. It removes the following case:
* no semver key in comment & no semver label on pr.

Reply:
https://github.com/myakove/pr-workflows/pull/33#discussion_r701350472